vagrant-default.yml

```diff
@@ -522,3 +522,6 @@ redis:
         pidfile: /var/run/redis/redis-server.pid
         port: 6379
         unixsocket: /var/run/redis/redis.sock
+
+bindmounts: []
```

Comment:
Hello @dmaletta thanks for the contribution.

I would prefer having this option somewhere on the top, in front of: https://github.com/vagrant-php/ubuntu/blob/18.04/vagrant-default.yml#L32